src/azul/service/responseobjects/elastic_request_builder.py

```diff
@@ -101,9 +101,10 @@ def create_query(filters):
         return Q('bool', must=query_list)
 
     @staticmethod
-    def create_aggregate(filters, facet_config, agg):
+    def create_aggregate(filters, facet_config, agg, request_config):
         """
         Creates the aggregation to be used in ElasticSearch
+        :param request_config: Dictionary mappings for internal and external reference
```

Comment:
This sentence makes little sense to me. A dictionary IS a mapping. Not sure what you mean by "internal and external reference".